Minor doc improvements. (#141)
crc-org/vfkit: doc/missing-vz-api.md

# Missing APIs

> This document contains a non-exhaustive list of APIs not currently used/supported by `vfkit`, but could be useful to have.

Reviewer: "but could be useful to have", or "but which could be useful to have"? The latter makes more sense to me, but I'm not a native English speaker.

Reviewer: (and I suspect this doc is out of date by now, I don't think I've updated it after macOS 14 release)

Author: ```suggestion
This document is a non-exhaustive list of unimplemented APIs, which may be worth adding to `vfkit`.
```

How's this?
